### PR TITLE
Glossary created. First batch of terms added.

### PR DESCRIPTION
**Pull request by linnnsss**

### CODE REVIEW

File: docs/glossary/terms.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+---
+sidebar_position: 1
+sidebar_label: Terms
+---
+
```
Comment (ShookLyngs) [inline docs/glossary/terms.md:5]: Maybe add a title? something like: "# Terms"

File: docs/glossary/terms.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+## anyone-can-pay (ACP)
```
Comment (ShookLyngs) [inline docs/glossary/terms.md:6]: Why are the initial letters of the terms all in lowercase? Should they be capitalized instead?
For example: "anyone-can-pay (ACP)" -> "Anyone-can-pay (ACP)"

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: docs/glossary/terms.md (hunk 2 above)
Comment (linnnsss) [reply docs/glossary/terms.md:7]: Regarding capitalization there's no strict rule. The key is to keep the style consistent.

File: docs/glossary/terms.md (hunk 1 above)
Comment (linnnsss) [reply docs/glossary/terms.md:6]: > Maybe add a title? something like: "# Terms"

added

**Review by ShookLyngs — APPROVED** [review]

File: docs/glossary/terms.md (hunk 2 above)
Comment (anipenguin) [reply docs/glossary/terms.md:7]: I vote for capitalizing the first letter of every term, the current mix is awkward

**Comment by anipenguin** [comment]
Can we put the types of lock scripts under **lock script**? It’s quite overwhelming for the glossary to start with ACP assuming people know what a CKB lock script is as well as concepts like "Simple UDT"

File: docs/glossary/terms.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+## private key
+
+Cryptographic information exclusively known to its owner or owners. Employed in conjunction with an algorithm for both encryption and decryption, ensuring the confidentiality and security of sensitive information.
+
+**see also:** 
+* [public key](#public_key)
+
+## public cluster
+
+Cluster enabling anyone, besides the owner, to associate a Spore or multiple Spores when minting.
```
Comment (anipenguin) [inline docs/glossary/terms.md:170]:
```suggestion
A cluster that allows anyone, not just the owner, to associate a Spore or multiple Spores with it when minting Spores.
```
"besides the owner" excludes the owner

File: docs/glossary/terms.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+**see also:**
+* [Why is Spore‘s value redeemable?](https://docs.spore.pro/basics/tech-faq#why-is-spores-value-redeemable)
+
+## secp256k1-blake160
+
+A lock script code using the [secp256k1](https://en.bitcoin.it/wiki/Secp256k1) signature algorithm, performing [Blake2b](https://www.blake2.net/) hash operations on transaction and witness components. Essential for validating transactions in specific blockchain systems.
+
+## transfer
+
+Process that refreshes a Spore by consuming its cell and creating a new one with the same Spore ID and unchanged fields. This operation costs no additional free, where the costs is deducted from the cell's capacity.
```
Comment (anipenguin) [inline docs/glossary/terms.md:196]:
```suggestion
The process of changing the ownership of a Spore or a Cluster. It's important to note that owning a cluster does not automatically grant ownership of the spores associated with that cluster. Each Spore and Cluster has its own distinct ownership rights that can be transferred separately.
```
the implementation bits should be sufficient under "see also", especially the information is the same as basics/spore-operation#transfer

**Review by anipenguin — CHANGES_REQUESTED** [review]

**Review by linnnsss — COMMENTED** [review]
items in all capitalized; relocated anyone-can-pay, omnilock, secp256k1-blake160 under lock script

File: docs/glossary/terms.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+**see also:** 
+* [cluster](#cluster)
+
+## immortal
+
+Extension making a spore or a cluster permanently indestructible on-chain, impossible to be destroyed or remove from the chain, even by their owners.
+
+## immutability
+
+Quality of permanence and resistance to alteration. Once a Spore is minted, its essential attributes, including ownership, provenance, and metadata, become fixed and cannot be changed. This guarantees the security and integrity of these attributes on the blockchain.
```
[the remark this answers is not in the document]
Comment (anipenguin) [reply docs/glossary/terms.md:73]:
```suggestion
Quality of permanence and resistance to alteration. Once a Spore is minted, its essential attributes become fixed and cannot be changed. 
```

**Review by anipenguin — APPROVED** [review]

**Pull request merged**